### PR TITLE
Initial version of new emails for GLAD alerts

### PR DESCRIPTION
**Pull request by ikas**
This PR updates the GLAD alerts email publisher to take into the new layouts and new datasets for GLAD alerts.

The designs for the emails can be found here: https://basecamp.com/3063126/projects/10727890/todos/393337827?enlarge=392520529#attachment_392520529

Before merging this PR, please merge this Areas PR: https://github.com/gfw-api/gfw-area/pull/66

### CODE REVIEW

File: app/src/presenters/gladPresenter.js [hunk 1]
```diff
@@ -1,8 +1,67 @@
+const _ = require('lodash');
 const logger = require('logger');
 const moment = require('moment');
 const config = require('config');
 const ctRegisterMicroservice = require('ct-register-microservice-node');
 
+function average(data) {
```
Comment (tiagojsag) [inline app/src/presenters/gladPresenter.js:7]: https://lodash.com/docs/4.17.15#mean

File: app/src/publishers/emailPublisher.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ const TEMPLATE_MAP = {
     'forma-alerts': 'forest-change-notification',
     forma250GFW: 'forest-change-notification',
     'terrai-alerts': 'forest-change-notification',
-    'glad-alerts': 'forest-change-notification'
+    'glad-alerts': 'forest-change-notification-no-chart-copy'
```
Comment (edbrett) [inline app/src/publishers/emailPublisher.js:12]: this is the wrong template! It should be the same as before.

Comment (edbrett) [reply app/src/publishers/emailPublisher.js:12]: Sorry it should be `forest-change-notification-glads-{{lang}}`

**Comment by edbrett** [comment]
And one last request if you have the time. Would it be possible to update all links in emails to also send the language code as a query parameter? For example `https://gfw.org/area/id?lang={lang_code}`

File: app/src/publishers/emailPublisher.js (hunk 2 above)
Comment (ikas) [reply app/src/publishers/emailPublisher.js]: Fixed :+1:

**Comment by ikas** [comment]
> Could we also make sure any reference/link from the API that goes to `/my-gfw/subscriptions` or similar goes to `/my-gfw`

Done :+1:

**Comment by ikas** [comment]
> And one last request if you have the time. Would it be possible to update all links in emails to also send the language code as a query parameter? For example `https://gfw.org/area/id?lang={lang_code}`

Done too :+1:

**Pull request merged**